digitalpublicgoods/flow-cameroon.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Flow Cameroon",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/Idadelveloper/Flow/blob/main/README.md"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "Yes",
+    "pltIndependenceDesc": "Flow Cameroon requires Firebase and Firestore which are closed dependencies. However other open source alternatives exists such Couchbasev (https://www.couchbase.com/)"
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/Idadelveloper/Flow/blob/main/README.md"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "There is no legislation for data protection in Cameroon. Here is our privacy policy: https://dscuba2020.github.io/projects/flow-website/privacy%20policy.html"
+    ],
+    "adherenceSteps": [
+      "Privacy policy - https://dscuba2020.github.io/projects/flow-website/privacy%20policy.html"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "Data Exchange formats - JSON",
+      "Web standards - HTML, CSS, ECMAScript",
+      "Internationalization (i18n) - UTF-8",
+      "Authentication & Authorization - OAuth 2, OpenID Connect"
+    ],
+    "evidenceStandardSupport": [
+      "https://github.com/bernandez/Flow"
```

Comment:
What relationship exists between this repository and the one provided for `documentationURL` above? One seems a fork of the other; if that's the case, only one should be provided.

Reply:
Resolved in commit 9e47afd